description.md

```diff
@@ -18,8 +17,34 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Assignments
 
-### Drills
+Assignments are practical activities that learners undertake individually or in teams over the span of multiple weeks.
+The primary objective of an assignment is to create a compelling scenario that allows learners to apply the concepts they have learned.
+
+#### Support
+
+Assignments are typically meant for individual work, but they often require support from trainers.
+This support aims to clarify requirements and prevent students from getting stuck on tasks.
+
+Going beyond basic assistance, forums can be implemented as public platforms where learners can openly share questions, provide answers, and engage in discussions without revealing explicit solutions.
```

Comment:
You should touch on our plan to have students review each other's work.

Reply:
We have seen more success with direct feedback on live chats, like Teams. The advantage of this is that it's more of a conversation, with the disadvantage that threads might get lost in the chatter.

Maybe we should tough on this too. Only forums for support is restrictive.

Reply:
> You should touch on our plan to have students review each other's work.

This makes sense in the context of smaller tasks like drills or lab exercises. 
The goal is to have assignments automatically graded, making students review each other's work does little to improve the general experience.